pages/api/users/get-social.js

```diff
@@ -0,0 +1,49 @@
+import * as Data from "~/node_common/data";
+import * as Serializers from "~/node_common/serializers";
+
+export default async (req, res) => {
+  const subscriptions = await Data.getSubscriptionsByUserId({ userId: req.body.data.userId });
+
+  if (!subscriptions) {
+    return res.status(404).send({ decorator: "SERVER_USER_SUBSCRIPTIONS_NOT_FOUND", error: true });
+  }
+
+  if (subscriptions.error) {
+    return res.status(500).send({ decorator: "SERVER_USER_SUBSCRIPTIONS_NOT_FOUND", error: true });
+  }
+
+  const subscribers = await Data.getSubscribersByUserId({ userId: req.body.data.userId });
+
+  if (!subscribers) {
+    return res.status(404).send({ decorator: "SERVER_USER_SUBSCRIBERS_NOT_FOUND", error: true });
+  }
+
+  if (subscribers.error) {
+    return res.status(500).send({ decorator: "SERVER_USER_SUBSCRIBERS_NOT_FOUND", error: true });
+  }
+
+  let serializedUsersMap = { [req.body.data.userId]: req.body.data };
+  let serializedSlatesMap = {};
+
+  const r1 = await Serializers.doSubscriptions({
+    users: [],
+    slates: [],
+    subscriptions,
+    serializedUsersMap,
+    serializedSlatesMap,
+  });
+
+  const r2 = await Serializers.doSubscribers({
```

Comment:
In the future when people are copy & pasting some of my code, it might be better to be more redundant, this reads like a sentence:

`const serializedSubscribers = await Serializers.doSubscribers();`

This is my fault, `r2` is a really bad variable name. But for the future, let's move towards naming things where people can read the behavior like they were reading a text message (within reason).